lib/new_relic/agent/instrumentation/memcache.rb

```diff
@@ -2,7 +2,7 @@
 # See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
 # frozen_string_literal: true
 
-# NOTE there are multiple implementations of the MemCache client in Ruby,
+# NOTE: there are multiple implementations of the MemCache client in Ruby,
 # each with slightly different API's and semantics.
```

Comment:
I saw the "MemCache" spelling of Memcached and raised an eyebrow. Then I got to the apostrophe in "API's"  and now I'm out.

Reply:
I don't know if I can in good conscience walk away from this.